add-on/chrome/modules/MozLoopAPI.jsm

```diff
@@ -960,58 +919,6 @@ const kMessageHandlers = {
   },
 
   /**
-   * Starts alerting the user about an incoming call
-   *
-   * @param {Object}   message Message meant for the handler function, containing
-   *                           the following parameters in its `data` property:
-   *                           [ ]
-   * @param {Function} reply   Callback function, invoked with the result of this
-   *                           message handler. The result will be sent back to
-   *                           the senders' channel.
-   */
-  StartAlerting: function(message, reply) {
-    let chromeWindow = Services.wm.getMostRecentWindow("navigator:browser");
-    chromeWindow.getAttention();
-    ringer = new chromeWindow.Audio();
-    ringer.src = Services.prefs.getCharPref("loop.ringtone");
```

Comment:
Please remove from prefs.js, and also remove the ringtone.ogg from the repo & jar.mn etc.